core/templates/dev/head/pages/profile/Profile.js

```diff
@@ -109,13 +105,10 @@ oppia.controller('Profile', [
       };
 
       $scope.updateSubscriptionButtonPopoverText = function() {
-        if ($scope.userNotLoggedIn === true) {
+        if ($scope.userNotLoggedIn) {
           $scope.subscriptionButtonPopoverText = ('Log in or sign up to ' +
```

Comment:
The correct formatting for this is:

```
  $scope.subscriptionButtonPopoverText = (
    'Log in or sign up to subscribe to your favorite creators.');
```

Please fix, here and below. (I.e., break at '(' and indent by two spaces for continuation lines.)